doc/nrf/drivers/nrf700x_low_api_wifi.rst

```diff
@@ -0,0 +1,11 @@
+Normal operation
```

Comment:
`Wi-Fi mode` instead of `Normal operation`. Also we can move this above the `Radio Test mode`